### PR TITLE
Update poetry.lock query

### PR DESCRIPTION
**Pull request by jdorfman**
Signed-off-by: Justin Dorfman <justindorfman@gmail.com>

### Test

1. Ensure prettier has standardized the proposed changes.

### CODE REVIEW

File: content/blogposts/2022/openssl-307-vulnerabilities.md [hunk 1]
```diff
@@ -53,10 +53,10 @@ Ruby's OpenSSL gem still uses OpenSSL 1.1, so Ruby users (should) have nothing t
...
+However, the popular cryptography package does ship static OpenSSL libraries in many of its wheels, and versions 36.0.0 to 38.0.2 (inclusive) are vulnerable. (They are [about to release version 38.0.3](https://github.com/pyca/cryptography/issues/7758) with OpenSSL 3.0.7 bundled.) [This search](https://sourcegraph.com/search?q=context:global+%28f:requirements.*txt+cryptography%28%5Cs*%5B%3D%7E%5D%3D%5Cs*%2836%5C.%7C37%5C.%7C38%5C.0%5C.%5B0-2%5D%29%29%29+OR+%28f:poetry.lock+name%5Cs*%3D%5Cs*%22cryptography%22%5Cnversion%5Cs*%3D%5Cs*%22%2835%5C.%7C39%5C.%7C40%5C.0%5C.%5B0-2%5D%29%29&patternType=regexp) should cover most uses via pip, pipenv, and poetry:
 
 ```regex
-(f:requirements.*txt cryptography(\s*[=~]=\s*(36\.|37\.|38\.0\.[0-2]))) OR (f:poetry.lock name\s*=\s*"cryptography" AND version\s*=\s*"(36\.|37\.|38\.0\.[0-2]))
+(f:requirements.*txt cryptography(\s*[=~]=\s*(36\.|37\.|38\.0\.[0-2]))) OR (f:poetry.lock name\s*=\s*"cryptography"\nversion\s*=\s*"(35\.|39\.|40\.0\.[0-2]))
```
Comment (Joelkw) [inline content/blogposts/2022/openssl-307-vulnerabilities.md:59]: @jdorfman I don't know rust but isn't this bug present in lines 70-73 as well, or is that different? (E.g. not forcing these target to be on following lines with \n, but just in the same file with AND)

**Review by Joelkw — COMMENTED** [review]
Did we fix this in all places? If this is the only place that needs fixing, LGTM!

File: content/blogposts/2022/openssl-307-vulnerabilities.md (hunk 1 above)
Comment (jdorfman) [reply content/blogposts/2022/openssl-307-vulnerabilities.md:59]: Good catch!

**Review by Joelkw — APPROVED** [review]
Thanks for updating!

**Pull request merged**